btcwrapper/zmq/client.go

```diff
@@ -21,6 +21,8 @@ var (
 // Client manages ZMQ subscriptions and communication with a Bitcoin node.
 // It handles ZMQ message routing and provides thread-safe access to subscriptions.
 // Must be created with New() and cleaned up with Close().
+//
+//nolint:govet
```

Comment:
Can we disable only fieldalignment? Maybe `nolint:govet:fieldalignment`, try to google

Reply:
I found it, here is an example:
```
//nolint:errcheck(goconst)
```

Reply:
Or:
```
//nolint:govet/fieldalignment
```

Reply:
These doesnt work, I couldnt find how we can only disable the fieldalignment.